### DISCUSSION TITLE
discharging not working properly

### DISCUSSION THREAD

**Discussion by manuhank** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#854] [General]
Aldente is keeping the battery fully charged even if its well below the threshold, and with the discharge option on.
<img width="1484" alt="image" src="https://user-images.githubusercontent.com/19393548/207695777-08b7d6cf-8d9f-4f74-9536-dc991d0e652a.png">
<img width="439" alt="image" src="https://user-images.githubusercontent.com/19393548/207696209-823083d2-ef46-40c3-8c92-c59ea1e0284f.png">
version: 1.19.2
macos version: 13.0.1
has someone else expereicned this?

**Comment by MatthiasKerbl** [comment]
Hi @manuhank,

Did you do a macOS update recently?

Cheers,
Matthias

**Reply by manuhank** [reply]
My last update was back in october when ventura was first released. It worked well since then, although I remember having this same issue prior to the update.

**Reply by MatthiasKerbl** [reply]
That's unusual, normally, this kind of behavior happens after a macOS update.
I would still recommend the following steps. It might work again afterward:
1. Unplug and discharge your MacBook to less than your current charge limit in AlDente
2. Turn off your MacBook and close the lid for a couple of minutes
3. Unplug the power brick from the outlet and wait a couple of minutes
4. Turn on your MacBook and plug it in again
5. Check if it stops charging at the charge limit set in AlDente

**Reply by manuhank** [reply]
ok, I have upgraded to Ventura 13.1 and now everything is working as expected. That being said, I think this will happen again, as it has been doing, and I also think this might be related to switching between multiple users

**Reply by MatthiasKerbl** [reply]
The issue after a macOS update is known for a while now but it is extremely hard to fix because it is actually caused by macOS and not AlDente. We will post updates and news about this issue here https://github.com/davidwernhart/AlDente-Charge-Limiter/issues/768
If you want to use AlDente and fast user switching, please take a look at the following tutorial on our blog: https://apphousekitchen.com/fast-user-switching-explanation-and-tutorial/

Cheers,
Matthias

**Reply by manuhank** [reply]
I have installed aldente pro in both accounts. Plus, this happens randomly, and thats make me think is a bug.
thanks for your time. I guess Ill come back if something new comes.
